buildconfig/manylinux-build/build-wheels.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 set -e -x
 
-export SUPPORTED_PYTHONS="cp27-cp27mu cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39"
+if [ `uname -m` == "aarch64" ]; then
+   export SUPPORTED_PYTHONS="cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39" 
```

Comment:
I guess because the manywheel images don't support python 2? It's something we can improve on later if anyone wants to contribute it.